### DISCUSSION TITLE
Removing autofocus on popup form

### DISCUSSION THREAD

**Discussion by ronamisterbit** [elementor/elementor#23164] [Developers Community]
Hey community,
I have a problem in which a field in a form I created inside a popup gets autofocused, and on mobile it brings the keyboard up and just looks bad. I want to remove the autofocus entirely. 
I think this should be a feature but unfortunately it is not, and Elementor support could not help me with this as well.

I guess i can achieve this with some kind of custom code.
Does anybody have any idea how to achieve this, while maintaining the functionality of the form?

Thanks
Ron.

**Comment by loucasl** [comment]
This is likely because you have the Accessible Navigation toggle turned on in the popup settings, if you turn it off, the focus should disappear

**Reply by ronamisterbit** [reply]
Omg it worked! thank you so much. 
So weird i couldn't find this solution anywhere online!

**Reply by Daveden2** [reply]
I definitely wouldn't recommend turning the accessibility toggle off in the popup. That's bad for accessibility reasons (as the description suggests) 

Instead, I'll suggest you add a button or focusable element as the 1st item and hide it using "sr-only" custom css.

I did a video about it a while ago

**Reply by zine-eddine27** [reply]
Wow thank you , really tricky !

**Reply by rosskevin** [reply]
@Daveden2 - super awesome you did a video.  Unfortunately, for history's sake, it is not linked to this issue which is discoverable by google.

**Reply by Daveden2** [reply]
@rosskevin, I'm not sure I understand. What issue are you referring to? Is it the autofocus on form fields?

**Reply by rosskevin** [reply]
@Daveden2 you said "I did a video about it a while ago" above but didn't link to the video.  I was curious how you removed the autofocus.

**Reply by toineenzo** [reply]
> @Daveden2 you said "I did a video about it a while ago" above but didn't link to the video.  I was curious how you removed the autofocus.

@rosskevin Check his video out here
https://youtu.be/3pfPYhgmAZU
